cypress/integration/AppAnalytics/DiseaseOnly/DiseaseAnalytics.feature

```diff
@@ -22,3 +22,19 @@ Feature: Disease listing page analytics
 			| page.additionalDetails.diseaseName          | breast cancer                                             |
 			| page.additionalDetails.trialListingPageType | disease                                                   |
 			| page.additionalDetails.numberResults        | (int)938                                                  |
+
+	Scenario: Click event fires when a user clicks on result item
+        Given "trialListingPageType" is set to "Disease"
+				And "dynamicListingPatterns" object is set to "Disease"
+       	And "pageTitle" is set to "{{disease_label}} Clinical Trials"
```

Comment:
Please remove this line, as setting the dynamicListingPatterns on the line above takes care of it. pageTitle being set is only necessary for Manual pages now.